### ISSUE TITLE
Escaping special characters in dictionary items

### ISSUE THREAD

**Issue by oguya** [oguya/cis-ubuntu-14-ansible#7]
**step to reproduce**
- code snippet from tasks/section-08-level-2.yml: 59
  
  ```
  - name: 8.1.4 Record Events That Modify Date and Time Information (Scored)
    lineinfile: dest=/etc/audit/audit.rules line={{ item }} insertafter=EOF state=present
    with_items:
      - "-a always,exit -F arch=b64 -S adjtimex -S settimeofday -k time-change"
      - "-a always,exit -F arch=b32 -S adjtimex -S settimeofday -S stime -k time-change"
      - "-a always,exit -F arch=b64 -S clock_settime -k time-change"
      - "-a always,exit -F arch=b32 -S clock_settime -k time-change"
      - "-w /etc/localtime -p wa -k time-change"
    when: ansible_machine == "x86_64"
    notify: restart auditd
    tags:
      - scored
      - section8.1.4
  ```
- error when ansible
  
  ```
  $ ansible-playbook cis.yml -K --tags=section8.1.4 --check
  SUDO password: 
  
  PLAY [all] ******************************************************************** 
  
  GATHERING FACTS *************************************************************** 
  ok: [cis01]
  
  TASK: [cis | 8.1.4 Record Events That Modify Date and Time Information (Scored)] *** 
  fatal: [cis01] => A variable inserted a new parameter into the module args. Be sure to quote variables if they contain equal signs (for example: "{{var}}").
  
  FATAL: all hosts have already failed -- aborting
  
  PLAY RECAP ******************************************************************** 
             to retry, use: --limit @/home/james/cis.retry
  
  cis01                      : ok=1    changed=0    unreachable=1    failed=0   
  ```

**Comment by oguya** [comment]
To fix this issue, we'll need to replace all `{{ item }}` with `"{{item}}"`. Infact if a variable contains a string with `=` or `spaces` then you should enclose the variable in double quotes(`"`)

**Comment by oguya** [comment]
I think this issue should be fixed in both commits 3e8fa25 and 032c9d3
